geocoding/clean.py

```diff
@@ -0,0 +1,30 @@
+import fire
+import pandas as pd
+
+
+def cleaner(excel_sheet):
+    """Will combine all the CSV's formed during the geocoding process
+    
+    Args:
+        excel_sheet (str): The excel sheet that was geocoded
+    """
+    temp = pd.ExcelFile(excel_sheet)
+    sheet_names = temp.sheet_names
+
+    maindf = pd.read_csv(f"Sheets/{sheet_names[0]}.csv")
```

Comment:
```suggestion
    main_df = pd.read_csv(f"Sheets/{sheet_names[0]}.csv")
```
1. Always use snake case variable naming please? Or use a linter like autoPEP8. 
2. Can we use more specific variables like like `xl_pd_buffer` (instead of `temp`) and `sheet_df` (instead of `main_df`) so that they are clearer?

Reply:
Got it. Renamed the variables.